### PR TITLE
FIX z-index of select dropdown clashing with TinyMCE toolbar

### PR DESCRIPTION
**Pull request by christopherdarling**
https://github.com/silverstripe/silverstripe-tagfield/issues/270

### CODE REVIEW

**Comment by GuySartorelli** [comment]
Please provide some explanation of what the specific problem is that this pr intends to resolve.

**Comment by christopherdarling** [comment]
@GuySartorelli please see the linked issue for context

**Comment by GuySartorelli** [comment]
If there is an issue that this pr relates to, please at the very very least link to it in the pr description as per [the documented contribution guide](https://docs.silverstripe.org/en/5/contributing/code/#create-the-pr)

**Comment by christopherdarling** [comment]
Sure all done

File: client/src/components/TagField.scss [hunk 1]
```diff
@@ -108,3 +108,7 @@
     color: #0071e6;
   }
 }
+
+.ss-tag-field__menu {
+  z-index: 10;
```
Comment (ChloeSartorelli) [inline client/src/components/TagField.scss:115]:
```suggestion
  // Prevents the dropdown from hiding behind TinyMCE
  // see https://github.com/JedWatson/react-select/issues/5329
  z-index: 10;
```
Please add this so there's some immediate feedback about why this is needed - it's pretty weird to need a z-index like this so we're bound to want to know about it in the future.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Looks good, works locally. Just one small change.

If you want this to be available immediately as a patch release, please retarget the PR to the `3.1` branch (you may need to [reset your commits](https://gist.github.com/GuySartorelli/74627843aeb75e12b777b1f44cb22d2f) afterward).

**Pull request closed without merging**